packages/select/src/index.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import {useDefaults} from '@smashing/theme'
+import {SelectProps} from './types'
+import {S} from './styles'
+import styled from 'styled-components/macro'
+import {ButtonAppearanceType, ButtonIntentType} from '@smashing/button'
+
+const SelectFC: React.FC<SelectProps> = ({children, ...props}) => {
+  const defaults = useDefaults<SelectProps>('select', props, {
+    options: [],
+    value: '',
+    onChange: () => undefined,
+    height: 32
+  })
+
+  const internalOnChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
We're not processing change event in other input components. I would just pass `props.onChange` to `S.SelectButtonComponent`